source/parameters/matter.md

```diff
@@ -0,0 +1,10 @@
+(s:matter.params)=
+## MatterP:  Definition of materials
```

Comment:
I would propose to add the  `state` and `matter:` (formerly: `material:`) properties here and remove the `Material` concept.